src/qtt/utilities/tools.py

```diff
@@ -971,7 +1000,8 @@ def addPPTslide(title=None, fig=None, txt=None, notes=None, figsize=None,
         else:
             slide.shapes.title.textframe.textrange.text = 'QCoDeS measurement'
 
-        from qtt.measurements.videomode import VideoMode  # import here, to prevent default imports of gui code
+        from qtt.measurements.videomode import \
```

Comment:
Why the line break? Line was not too long.
Why break the line here instead of sprint the comment from the code
```
# import here, to prevent default imports of gui code
from qtt.measurements.videomode import VideoMode

```